lib/fpm/command.rb

```diff
@@ -212,6 +212,12 @@ def help(*args)
         "Currently only supports deb, rpm and pacman packages." do |val|
     File.expand_path(val) # Get the full path to the script
   end # --before-upgrade
+  option "--triggered", "FILE",
```

Comment:
The trigger support here seems to be targeted specifically at debian packages, so it feels best to move this option to the deb target instead of the global command line flags. Thoughts?

Reply:
rpm implements triggers too (http://ftp.rpm.org/api/4.4.2.2/triggers.html) but they are different.
Thus I'll change it to `--deb-triggered`.